models/pardot__prospects.sql

```diff
@@ -8,10 +8,20 @@ with prospects as (
     select *
     from {{ ref('int__activities_by_prospect') }}
 
+), enriched_prospects as (
+
+    select 
+        prospect_id,
+        account_id,
+    from {{ ref('int_pardot__prospects_join_enriched') }}
+
 ), joined as (
 
     select
         prospects.*,
+
+        enriched_prospects.account_id,
+
         {% for event_type in var('prospect_metrics_activity_types') %}
```

Comment:
More of a clarifying question, but are we using the default `email` and `visit` event types, or are we incorporating other `activity_type`s in the variable somewhere?